integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -1424,6 +1424,162 @@ func TestVRFV2PlusWithBHS(t *testing.T) {
+		subIDsForCancellingAfterTest = append(subIDsForCancellingAfterTest, subIDs...)
+
+		_, err = consumers[0].RequestRandomness(
+			vrfKey.KeyHash,
+			subID,
+			*configCopy.VRFv2Plus.General.MinimumConfirmations,
+			*configCopy.VRFv2Plus.General.CallbackGasLimit,
+			isNativeBilling,
+			*configCopy.VRFv2Plus.General.NumberOfWords,
+			*configCopy.VRFv2Plus.General.RandomnessRequestCountPerRequest,
+		)
+		require.NoError(t, err, "error requesting randomness")
+
+		randomWordsRequestedEvent, err := vrfContracts.CoordinatorV2Plus.WaitForRandomWordsRequestedEvent(
+			[][32]byte{vrfKey.KeyHash},
+			[]*big.Int{subID},
+			[]common.Address{common.HexToAddress(consumers[0].Address())},
+			time.Minute*1,
+		)
+		require.NoError(t, err, "error waiting for randomness requested event")
+		vrfv2plus.LogRandomnessRequestedEvent(l, vrfContracts.CoordinatorV2Plus, randomWordsRequestedEvent, isNativeBilling)
+		randRequestBlockNumber := randomWordsRequestedEvent.Raw.BlockNumber
+		var wg sync.WaitGroup
+		wg.Add(1)
+		//Wait at least 260 blocks
+		_, err = actions.WaitForBlockNumberToBe(randRequestBlockNumber+uint64(260), evmClient, &wg, time.Second*262, t)
+		wg.Wait()
+		require.NoError(t, err)
+		configCopy.VRFv2Plus.General.SubscriptionFundingAmountNative = ptr.Ptr(float64(1)) // 1 ETH
+		l.Info().Float64("SubscriptionFundingAmountNative", *configCopy.VRFv2Plus.General.SubscriptionFundingAmountNative).
+			Float64("SubscriptionFundingAmountLink", *configCopy.VRFv2Plus.General.SubscriptionFundingAmountLink).
+			Msg("Funding subscription")
+		err = vrfv2plus.FundSubscriptions(
+			env,
+			chainID,
+			big.NewFloat(*configCopy.VRFv2Plus.General.SubscriptionFundingAmountNative),
+			big.NewFloat(*configCopy.VRFv2Plus.General.SubscriptionRefundingAmountLink),
+			vrfContracts.LinkToken,
+			vrfContracts.CoordinatorV2Plus,
+			subIDs,
+		)
+		require.NoError(t, err, "error funding subscriptions")
+		randomWordsFulfilledEvent, err := vrfContracts.CoordinatorV2Plus.WaitForRandomWordsFulfilledEvent(
+			[]*big.Int{subID},
+			[]*big.Int{randomWordsRequestedEvent.RequestId},
+			time.Minute*2,
+		)
+		require.NoError(t, err, "error waiting for randomness fulfilled event")
+		vrfv2plus.LogRandomWordsFulfilledEvent(l, vrfContracts.CoordinatorV2Plus, randomWordsFulfilledEvent, isNativeBilling)
+		status, err := consumers[0].GetRequestStatus(testcontext.Get(t), randomWordsFulfilledEvent.RequestId)
+		require.NoError(t, err, "error getting rand request status")
+		require.True(t, status.Fulfilled)
+		l.Debug().Bool("Fulfilment Status", status.Fulfilled).Msg("Random Words Request Fulfilment Status")
+
+		clNodeTxs, _, err := nodeTypeToNodeMap[vrfcommon.BHF].CLNode.API.ReadTransactions()
+		require.NoError(t, err, "error fetching txns from BHF node")
+		batchBHSTxFound := false
+		for _, tx := range clNodeTxs.Data {
+			if strings.EqualFold(tx.Attributes.To, vrfContracts.BatchBHS.Address()) {
+				batchBHSTxFound = true
+			}
+		}
+		require.True(t, batchBHSTxFound)
```

Comment:
nit - ideally, we would want to get the actual tx and decode input data like [here](https://github.com/smartcontractkit/chainlink/blob/9d6c894ade1d38fb8312552b6a03836b7218136d/integration-tests/smoke/vrfv2plus_test.go#L1409), but not necessary